internal/bminventory/inventory.go

```diff
@@ -1160,6 +1160,11 @@ func (b *bareMetalInventory) InstallCluster(ctx context.Context, params installe
 			fmt.Sprintf("Updated status of cluster %s to installing", cluster.Name), time.Now())
 	}()
 
+	// Delete previews installation log files from object storage, if exist.
+	if err := b.clusterApi.DeleteClusterLogs(ctx, &cluster, b.objectHandler); err != nil {
```

Comment:
Can u please add delete logs_collected_at flag too? Lets clearer all the logs relevant stuff before starting installation